src/rpms.sh

```diff
@@ -497,8 +546,9 @@ rlCheckMakefileRequires() {
 
 =head3 rlAssertRequired
 
-Ensures that all Requires, specified in beakerlib-style beaker-wizard layout
-Makefile, are installed.
+Ensures that all requires and recommends specified in the metadata.yaml are installed.
```

Comment:
Should `rlAssertRequired` by default assert recommended packages as well? I guest this might be a bit confusing. I would expect that recommended packages (by definition) do not have to be installed on the system.

Reply:
Yes, but originally the Makefile's _Requires_ and _RhtsRequires_ are now basically _recommend_ and _require_, respectively. That's the reason I included both.

Reply:
Oh, I see. Thanks for clarification. Let's merge this! :-)